### DISCUSSION TITLE
♿ Add "Accessible Navigation" toggle in the Off-Canvas widget similar to Popups

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#27887] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi elementor team,

Currently, if we have an input like a form element inside the offcanvas, the keyboard is activated on mobile immediately after the offcanvas is opened, and this is very annoying.

### Describe the solution you'd like.

Please add the Accessible navigation option to the offcanvas element. We currently have this option in the popup.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Daveden2** [comment]
Hey @79ho3ein, instead of trying to make your off-canvas widget keyboard inaccessible, why not add a close button as the first focusable element within it?
